### ISSUE TITLE
AT9S

### ISSUE THREAD

**Issue by luizanesio** [wdcossey/SharpPropoPlus#5]
Hi William! I looking to use my transmitter as a controller for MS Simulator 2020. Is this SharpPropoPlus working? I´m using the SmartPropoPlus + VJoy to use my AT9S RadioLink upgraded to work with 12 channels but the SmartPropoplus just work with the 10 first channels. Is SharpPropoPlus able to work with 12 channels? 
Can you build it for me because I dont know how to build it for myself?

**Comment by wdcossey** [comment]
@luizanesio 

I can have a look at adding 12 channels for your AT9 and making a build for you to test if you're unable to build it yourself?

**Comment by luizanesio** [comment]
Yes, William. I will appreciate your kindness very much. Thanks!

Regards!.

Luiz Anesio.


Em sex., 9 de out. de 2020 às 16:58, William David Cossey <
notifications@github.com> escreveu:
